src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private double radius;
+
+    public Circle(String color, double radius) {
+        super(color);
+        this.radius = radius;
+    }
+
+    @Override
+    public double getArea() {
+        return Math.PI * radius * radius;
+    }
+
+    @Override
+    public void draw() {
+        System.out.println("Figure: circle, area: "
+                + df.format(getArea()) + " sq. units, radius: "
+                + df.format(radius) + " units, color: "
+                + getColor());
+    }
+}
```

Comment:
It seems like the `Circle` class is extending `Figure` but does not provide a distinct behavior that would justify the use of inheritance. If `Figure` is an abstract class meant to share common behavior, ensure it's used correctly. If it's meant to define a contract for figures, an interface would be more appropriate.